### PR TITLE
Integrating Windows Testing Material

### PR DESCRIPTION
**Pull request by moxiegirl**
Integrating the Windows specific material into the contributor guide

Carries and Closes #11296 

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

**Comment by jfrazelle** [comment]
yayyyyy! thanks so much!

**Comment by jfrazelle** [comment]
:heart: 
:heart_eyes:

**Comment by icecrime** [comment]
Niiiice :-)

**Comment by moxiegirl** [comment]
ping @docker/docs-owners

File: docs/sources/project/test-and-docs.md [hunk 1]
```diff
@@ -230,6 +230,42 @@ with new memory settings.
 6. Restart your container and try your test again.
 
 
+## Testing just the Windows client
+
+This explains how to test the Windows client on a Windows server set up as a development environment.
+
+
+1.  If you don't have one, start a Git Bash window.
```
Comment (jamtur01) [inline docs/sources/project/test-and-docs.md]: What's a Git Bash window?

File: docs/sources/project/software-req-win.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+- a GitHub account
+- Git for Windows (msysGit)
+- TDM-GCC
+- MinGW (tar and xz)
+- Go language
+
+### Get a GitHub account
+
+To contribute to the Docker project, you will need a <a
+href="https://github.com" target="_blank">GitHub account</a>. A free account is
```
Comment (jamtur01) [inline docs/sources/project/software-req-win.md]: This seems to be italic for some reason? Perhaps GH parsing or perhaps an issue?

**Comment by jamtur01** [comment]
Looks like it needs a rebase.

File: docs/sources/project/test-and-docs.md (hunk 1 above)
Comment (moxiegirl) [inline docs/sources/project/test-and-docs.md]: image

Comment (jamtur01) [inline docs/sources/project/test-and-docs.md]: Sorry should have been clearer - we're using a proper noun to describe something - I wanted to confirm we explain how to do it or that is was a logical process for someone.

File: docs/sources/project/software-req-win.md (hunk 2 above)
Comment (moxiegirl) [inline docs/sources/project/software-req-win.md]: It maybe GH parsing, I'll check out when I rebase.

File: docs/sources/project/test-and-docs.md (hunk 1 above)
Comment (moxiegirl) [inline docs/sources/project/test-and-docs.md]: I think you are right, I moved this piece in and should have taken a look at that reference. I'll take care of it and resubmit.

File: docs/sources/project/software-req-win.md (hunk 2 above)
Comment (thaJeztah) [inline docs/sources/project/software-req-win.md]: It's GitHub confusing the underscore in `_blank` as Markdown for "italic", should be safe to ignore. (The rendered view on GitHub also looks okay)

Comment (moxiegirl) [inline docs/sources/project/software-req-win.md]: Looks fine in build:
image

**Comment by moxiegirl** [comment]
ping @docker/docs-owners

**Comment by jamtur01** [comment]
LGTM

**Comment by moxiegirl** [comment]
Author LGTM.

**Pull request merged**